pairing.py

```diff
@@ -5,64 +5,48 @@
 
 from random import choices
 
-class CollisionCounter:
+class Participant:
 
     def __init__(self, num_people, gamma=1/np.e):
```

Comment:
If this class = 1 individual participant, then I'm not too big of a fan of having each person have their own `num_people` and `gamma` parameters. To me it's a little confusing + implies that we could have separate gamma parameters for separate people. 

Maybe we just need a name that's something other than `Participant`

Reply:
Oh, yeah. I wasn't even thinking about `num_people`, but I realized that each person having their own `gamma` is redundant.

Reply:
I think a good quick fix for now is to just use a universal constant btw -- the `networkx` stuff and more indepth changes we can address later